template/build/webpack.prod.conf.js

```diff
@@ -33,6 +34,10 @@ var webpackConfig = merge(baseWebpackConfig, {
         warnings: false
       }
     }),
+    new OptimizeCssAssetsPlugin({
+       assetNameRegExp: /\.css$/,
+       cssProcessorOptions: { discardComments: { removeAll: true } }
```

Comment:
I *personally* believe that it is a best practice to leave in [`/*! important */` comments](https://github.com/twbs/bootstrap/blob/v4-dev/dist/css/bootstrap.css#L1) since that is typically where source name, version, copyright, & license is held:
* good for devs make it easier to find sources & versions if sourcecode isn't available
* lawyers & CEOs/CTOs need to know who owns what code of a project, especially when project ownership changes hands
* MIT & other OSS licenses tend to make it mandatory for "The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software."

So IMHO the default `{ discardComments: { removeAll: false } }` is better.  Hard to remember to flip over for lea